content/contributing/opencue/getting-started-windows.md

```diff
@@ -2,89 +2,131 @@
+
+   This adds an icon to the system tray.
+
+1. Right-click the pgAdmin icon and click **New pgAdmin window…**,
+   which will open  in your browser.
+
+1. Sign in with your PostgreSQL admin user credentials that you selected during
+   the PostgreSQL installation.
+
+1. To create a user, from the tree-view on the left, right-click
+   **Login/Group Roles** and select **Create→Login/Group Role…**.
+
+1. Name the user `opencue` and choose a secure password, then click **Save**.
+
+1. To create a database, from the tree-view on the left, right-click
+   **Databases** and select **Create** > **Database…**.
+   
+1. Name the database `opencue`, then click **Save**.
+
+   Next you need to populate the database.
+
+1. To populate the database, as described in the instructions on the
    [Setting up the Database](/docs/getting-started/setting-up-the-database/) 
    page to populate the database, you can:
-   1. Download the latest `schema-*.sql` from [the releases page](https://github.com/AcademySoftwareFoundation/OpenCue/releases).
+   
+   1. Download the latest `schema-*.sql` from
+      [the releases page](https://github.com/AcademySoftwareFoundation/OpenCue/releases).
+   
    1. In pgAdmin right-click the `opencue` database, and select **Query Tool…**.
-   1. In the Query Editor click the **📂 Open File** icon and open the `.sql` file you downloaded.
-   1. Click the **▶ Execute** icon to populate the database.
-1. **Insert Demo Data**: You will also need the demo data to run the full environment locally.
-   Download the latest `demo_data-*.sql` from the releases page and run it against the `opencue` database as in the previous step.
-1. **Grant Permissions**: In pgAdmin, right click on the `opencue` database and select **Grant Wizard**.
-   Use the select-all box to select all items, then select **Next**. Add the `opencue` user and select `ALL` privileges,
-   then select **Next** again, and finally **Finish**.
+   
+   1. In the Query Editor click 📂 (Open File icon) and open the `.sql` file you
+      downloaded.
+   
+   1. Click ▶ (Execute icon) to populate the database.
+
+   You also need the demo data to run the full environment locally.
+
+1. To insert demo data, download the latest `demo_data-*.sql` from the
+   [releases page](https://github.com/AcademySoftwareFoundation/OpenCue/releases)
+   and run it against the `opencue` database as described in the previous step.
+
+1. To grant permissions, in pgAdmin, right-click on the `opencue` database and
+   select **Grant Wizard**.
+
+1. Use the select-all box to select all items, then select **Next**.
+
+1. Add the `opencue` user and select `ALL` privileges, then select **Next**
+   again, and then click **Finish**.
 
 ## Configure Cuebot for local database access
 
-Add the database and user credentials into `cuebot/src/main/resources/opencue.properties`, e.g.:
+Add the database and user credentials into
+`cuebot/src/main/resources/opencue.properties`, as illustrated by the
```

Comment:
I know this isn't new content, but -- I wouldn't have them modify the default properties file. If they update their code they will either clobber those changes or won't be able to update.

Since they're using IntelliJ to run Cuebot, I would have them modify their Run Configuration to set these settings (i.e. by setting Program Arguments).

Reply:
Ping

Reply:
Apologies for the delay, I finally had some time to investigate this (Caveat: I'm not very familiar with IntelliJ or Windows).

It looks like we would want to [link to the IntelliJ docs](https://www.jetbrains.com/help/idea/creating-and-editing-run-debug-configurations.html?keymap=secondary_intellij_idea_classic), which are available with a variety of shortcuts, but only seem to offer macOS screenshots?

Then we'd need to update [the steps to run Cuebot](https://deploy-preview-153--elated-haibt-1b47ff.netlify.com/contributing/opencue/getting-started-windows/#running-cuebot). I don't have a way of testing any of this.

I could part of the way by trying to follow this on macOS, but I'm not sure about the syntax for the command to run Cuebot on Windows:

```
export JAR_PATH=<path to Cuebot JAR>
java -jar $JAR_PATH --datasource.cue-data-source.jdbc-url=jdbc:postgresql://$DB_HOST/$DB_NAME --datasource.cue-data-source.username=$DB_USER --datasource.cue-data-source.password=$DB_PASS
```

@Porges or @bcipriano would one of you be able to provide some brief notes with the Windows information I've outlined above (I'm assuming you both have access to Windows development environments?). I can then update the PR.

Alternatively, @bcipriano I think you can commit changes to this PR or @Porges we could submit this and you could follow up with another PR before we publish it?

Reply:
So instructions for how to edit the commandline in IntelliJ are:

Instead of the instruction that currently says to right-click `main` and select **Run**, select **Edit 'CubotApplicat....main()'...** and enter `--datasource.cue-data-source.jdbc-url=jdbc:postgresql://localhost/opencue --datasource.cue-data-source.username=opencue --datasource.cue-data-source.password=PASSWORD` into "Program arguments", and hit "OK". Finally **Run→Run 'CuebotApplication'**.

Reply:
Thanks @Porges! I should be able to get this updated over the next day or two.

Reply:
I've updated the guide:

https://github.com/AcademySoftwareFoundation/opencue.io/blob/windows-guide/content/contributing/opencue/getting-started-windows.md